<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chatRecordForm</class>
 <widget class="QWidget" name="chatRecordForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="msgBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>471</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="passwdlLayout">
    <item>
     <widget class="QLabel" name="passwdLabel">
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwdEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="passwdButton">
      <property name="text">
       <string>确定</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
